css/style.css

```diff
@@ -0,0 +1,589 @@
+    padding: 0;
+  }
+
+  #middle-navbar {
+    display: block;
+  }
+
+  .weather-icon {
+    width: 40px;
+  }
+
+  picture {
+    display: inherit;
+  }
+
+  /* This is for the middle navbar */
+
+  .nav-navigation {
+    display: flex;
+    justify-content: space-between;
+    height: 55px;
+  }
+
+  .navbar-nav1 {
+    display: flex;
+    flex-wrap: nowrap;
+    align-content: center;
+    align-items: stretch;
+    width: 90%;
+    height: 54px;
+  }
+
+  .navbar-nav1 li {
+    list-style-type: none;
+    padding: 3px;
+    flex-grow: 1;
+  }
+
+  ul.navbar-nav1 {
+    font-size: 1rem;
+    line-height: 24px;
+    font-weight: bold;
+    font-family: "Roboto Condensed", sans-serif;
+    color: black;
+  }
+
+  ul.navbar-nav1 li a {
+    color: black;
+    display: block;
+    padding: 15px 5px;
+    flex-grow: 1;
+  }
+
+  #nav-link-long {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .feature-section,
+  .story-section,
+  .debate-section {
+    height: auto;
+  }
+
+  .story-section,
+  .debate-section {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+  }
+
+  div.weather {
+    position: relative;
+    top: -90px;
+  }
+
+  .api {
+    z-index: 100;
+  }
+
+  nav.nav-navigation {
+    display: flex;
+  }
+
+  ul.navbar-nav {
+    font-size: 1.1rem;
+    line-height: 24px;
+    font-family: "Roboto Condensed", sans-serif;
+    color: black;
+  }
+
+  ul.navbar-nav1 li a,
+  ul.nav li {
+    color: black;
+    font-weight: 900;
+  }
+
+  ul.nav li,
+  ul.nav li a {
+    font: 18px/18px "Roboto Condensed", sans-serif;
+  }
+
+  ul.nav li a {
+    color: #807f84;
+    font-weight: 700;
+  }
+
+  ul.navbar-nav1 li a:hover,
+  ul.nav li a:hover {
+    color: #f72210;
+    text-decoration: none;
+  }
+
+  .debate-section {
+    position: -webkit-sticky;
+    position: sticky;
+    bottom: 1rem;
+    align-self: flex-start;
+  }
+
+  .email-subscription-top {
+    width: 80%;
+    height: auto;
+    display: inline;
+  }
+
+  .form-inline {
+    width: 14%;
+    justify-content: flex-start;
+  }
+
+  .form-control {
+    padding: 0;
+  }
+
+  .navbar-nav {
+    width: 86%;
+  }
+
+  #coll {
+    margin: 0;
+    display: flex;
+  }
+
+  .navbar-expand .navbar-collapse {
+    display: flex !important;
+  }
+
+  div.magazine-pick article > img,
+  div.editor-pick article > img {
+    width: 100%;
+  }
+
+  div.sub-left {
+    background-color: #f4f5f6;
+    height: 350px;
+  }
+
+  div.fn {
+    float: left;
+  }
+
+  div.f1 {
+    float: right;
+  }
+
+  div.fn,
+  div.fn.image {
+    width: 35%;
+    height: 0;
+  }
+
+  div.f1,
+  div.f1.info {
+    width: 65%;
+    margin-top: 10%;
+  }
+
+  div.mag-thumb.current {
+    z-index: 6;
+    top: -230px;
+    left: 100px;
+  }
+
+  div.last-section {
+    height: 200px;
+    margin: 0;
+  }
+
+  .feature-cont {
+    align-items: flex-start;
+  }
+
+  .story-section {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 4rem;
+    align-self: flex-start;
```

Comment:
- [ ] There is a bootstrap class for this behavior, please use bootstrap whenever it is possible.